pom.xml

```diff
@@ -3,18 +3,39 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <groupId>uk.gov.moj.cpp.common</groupId>
-        <artifactId>library-parent-pom</artifactId>
-        <version>2.0.0</version>
+        <groupId>uk.gov.justice</groupId>
+        <artifactId>maven-framework-parent-pom</artifactId>
+        <version>1.2.0</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>file-service</artifactId>
+    <groupId>uk.gov.justice.services</groupId>
+    <version>1.0.0-SNAPSHOT</version>
     <packaging>pom</packaging>
-    <version>1.0.0</version>
 
     <modules>
         <module>file-service-liquibase</module>
         <module>file-service-persistence</module>
+        <module>file-service-api</module>
+        <module>file-service-it</module>
     </modules>
+
+    <properties>
+        <common-bom.version>1.5.1</common-bom.version>
+        <cpp.framework.version>1.0.1</cpp.framework.version>
```

Comment:
Are we using this?

We should be including the framework-bon in a similar way to the common bom.. IF we need this...